<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlant</class>
 <widget class="QDialog" name="AddPlant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>818</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>v1.0</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"> 
QDialog{
color: rgb(255, 255, 255);
background-image: url(:/images/beauty.png);

}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>541</width>
     <height>513</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtScientificName">
      <property name="styleSheet">
       <string notr="true"> 
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Scientific name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtGenus">
      <property name="styleSheet">
       <string notr="true"> 
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Genus</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtCommonName">
      <property name="styleSheet">
       <string notr="true"> 
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Local name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtSpecies">
      <property name="styleSheet">
       <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Species</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtPrice">
      <property name="styleSheet">
       <string notr="true"> 
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Price ZMW</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPlainTextEdit" name="txtDescription">
      <property name="styleSheet">
       <string notr="true"> 
 QPlainTextEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cmbStatus">
      <property name="styleSheet">
       <string notr="true"> 
 QComboBox {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Status</string>
      </property>
      <item>
       <property name="text">
        <string>Available</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>On Nursery</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Out of Stock</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="txtQuantity">
      <property name="styleSheet">
       <string notr="true">
 QLineEdit {
  border:3px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
  font: 500 12pt ;
  color: rgb(0, 0, 0);;
}</string>
      </property>
      <property name="placeholderText">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>710</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton {
 
 font: 700 13pt ;
 color: rgb(255, 255, 255);
background-color: rgb(63, 63, 63);

}</string>
   </property>
   <property name="text">
    <string> Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>710</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton {
  
 font: 700 13pt ;
 color: rgb(255, 255, 255);
background-color: rgb(63, 63, 63);

}</string>
   </property>
   <property name="text">
    <string> Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_btn_home">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton {
  border:6px solid gray;
  border-radius: 10px;
  padding: 0 10px;
  background: grey;
 font: 700 12pt ;
 color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_image">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton {
  
 font: 700 12pt ;
 color: rgb(255, 255, 255);
background-color: rgb(63, 63, 63);

}</string>
   </property>
   <property name="text">
    <string>Browse image</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_field">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>451</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
 QLabel {
  border:2px solid gray;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
 
};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16777215</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  QLabel {
  background-color: rgb(0, 85, 0);
  border:6px solid green;
  border-radius: 10px;
  padding: 0 10px;
  
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{

  color: rgb(255, 255, 255);
  font: 800 13pt;
  border:3px solid white;
  border-radius: 10px;
  padding: 0 10px;

}</string>
   </property>
   <property name="text">
    <string> Rose Garden Nursery</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>40</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>40</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximize </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QLabel {
  color: rgb(255, 255, 255);
  font: 700 12pt ;
  border:1px solid  rgb(255, 255, 255);
  background-color: rgb(100, 100, 100);
  
}</string>
   </property>
   <property name="text">
    <string>    ADD PLANT | SECTION </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>160</y>
     <width>16777215</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	background-color: rgb(0, 98, 0);
    color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string>Normal Screen</string>
   </property>
  </widget>
  <zorder>formLayoutWidget</zorder>
  <zorder>btnSave</zorder>
  <zorder>btnReset</zorder>
  <zorder>btn_browse_image</zorder>
  <zorder>image_field</zorder>
  <zorder>label_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_btn_home</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label</zorder>
  <zorder>radioButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>AddPlant</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1373</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>1350</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddPlant</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1309</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>1264</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>AddPlant</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1384</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>1411</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
